util/is-inverse-benchmark-record.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param {{}} record
+ * @return {boolean}
+ */
+var isInverseBenchmarkRecord = function isInverseBenchmarkRecord(record) {
```

Comment:
I prefer using the named functions to help with debugging, but I've noticed that a lot of the newer environments seem to name anonymous functions for you based on their variable name or make it seem like they're named while you're debugging. So, it's probably not as compelling of a reason nowadays. 

Also, prefer putting function expressions up top over the declarations that hoist and are usually placed below where they're called, but I'm not overly opinionated on this. When in Rome.